src/test_snmp.cpp

```diff
@@ -502,3 +503,52 @@ TEST_F(SNMPTest, ContinuousAccumulatorTable)
   cwtest_reset_time();
   delete tbl;
 }
+
+TEST_F(SNMPTest, CxCounterTable)
+{
+  cwtest_completely_control_time();
+  
+  // Create table
+  SNMP::CxCounterTable* tbl = SNMP::CxCounterTable::create("cx_counter", test_oid);
+  
+  // Check that the rows that are there are the ones we expect and that initial
+  // values are zero.
+  FILE* fd = popen("snmpwalk -v2c -On -c clearwater 127.0.0.1:16161 .1.2.2", "r");
```

Comment:
Is it worth factoring this out into a function equivalent to `snmp_get`? I'm thinking of an `snmp_walk` function that returns a map of OID strings to ints - that might make the test clearer (and lets you check, for example, that your table has the expected number of rows by checking the size of the map).